### PR TITLE
Copy update on /16-04

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Copy update on /16-04
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare page against [copy doc](https://docs.google.com/document/d/1wclofqY9b3v_M9igpkYn6or5uXx0B_Xb_nVVzGBbOLY/edit#)
- Demo at https://ubuntu-com-11433.demos.haus/16-04

## Issue / Card

Fixes [#5016](https://github.com/canonical-web-and-design/web-squad/issues/5016)

### CODE REVIEW

**Review by sowasred2012 — APPROVED** [review]
LGTM! Just the tiniest tweak, and then good to merge

File: templates/16-04/index.html [hunk 1]
```diff
@@ -87,8 +87,8 @@ <h2>Ubuntu 16.04 LTS security updates</h2>
     <div class="col-6">
       <h2>Ubuntu 16.04 LTS Extended Security Maintenance (ESM)</h2>
       <h3>What is Ubuntu ESM?</h3>
-      <p><a href="/security/esm">Extended Security Maintenance</a> (ESM) provides extended Linux kernel and open source security updates for the Ubuntu base OS, key infrastructure components, like Ceph, OpenStack and Kubernetes, as well as open source applications, like PostgreSQL and NGINX.</p>
-      <p>ESM is available for free on up to three machines and for broader enterprise use through an Ubuntu Advantage subscription. The subscription includes additional services such FIPS-compliant modules and the Ubuntu Livepatch Service to apply critical kernel patches without unplanned downtime.</p>
+      <p><a href="/security/esm">Extended Security Maintenance</a> (ESM) provides extended Linux kernel and open source security updates for the Ubuntu base OS, key infrastructure components, like Ceph, OpenStack and Kubernetes, as well as open-source applications, like PostgreSQL and NGINX.</p>
```
Comment (sowasred2012) [inline templates/16-04/index.html:90]:
```suggestion
      <p><a href="/security/esm">Extended Security Maintenance</a> (ESM) provides extended Linux kernel and open-source security updates for the Ubuntu base OS, key infrastructure components, like Ceph, OpenStack and Kubernetes, as well as open-source applications, like PostgreSQL and NGINX.</p>
```

File: templates/16-04/index.html [hunk 2]
```diff
@@ -269,7 +245,7 @@ <h3 class="p-heading--4">Upgrading to Ubuntu 18.04 LTS, Ubuntu 20.04 LTS</h3>
     </div>
     <div class="col-6">
       <h3 class="p-heading--4">Upgrading Ubuntu 16.04 LTS on Azure, AWS</h3>
-      <p>If you are looking to upgrade your Ubuntu 16.04 LTS public cloud instance, it is recommended to launch new, Ubuntu Pro images. Ubuntu Pro for <a href="/azure/pro">Azure</a> and <a href="/aws/pro">AWS</a> are premium images with security and compliance features built in, and optimised for the public cloud.</p>
+      <p>If you are looking to upgrade your Ubuntu 16.04 LTS public cloud instance, it is recommended to launch new, Ubuntu Pro images. Ubuntu Pro for <a href="/azure/pro">Azure</a> and <a href="/aws/pro">AWS</a> are premium images with security and compliance features built-in, and optimised for the public cloud.</p>
       <p>Ubuntu Pro includes security coverage for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. Ideal for companies that have embraced open source for their new products.</p>
```
Comment (minkyngkm) [inline templates/16-04/index.html:249]: Hey @sowasred2012 this needs to be updated to open-source as well?

Comment (sowasred2012) [reply templates/16-04/index.html:249]: I thought this at first too, but according to [our copy style guide](https://docs.google.com/document/d/1AX-kSNztuAmShEoohe8L3LNLRnSKF7I0qkZGNeoGOok/edit#heading=h.ylqfftorfys4), "open source" should only be hyphenated when it's an adjective followed by a noun, so not in this case.

**Pull request merged**